Jakefile.js

```diff
@@ -976,6 +976,7 @@ var servicesLintTargets = [
 var lintTargets = compilerSources
     .concat(harnessCoreSources)
     .concat(serverCoreSources)
+    .concat(["client.ts"].map(function(f) { return path.join(serverDirectory, f); }))

```

Comment:
Oh, I didn't look there. I thought harness sources wouldn't include anything in the server directory. 

Adding it here is fine temporarily as long as we start linting `harnessSources` soon-ish. I tried adding it and it wasn't *too* bad. Can you create an issue and assign me, you or @weswigham to it? (Assuming Wesley is up for it next week.)